onnx/backend/test/loader/__init__.py

```diff
@@ -69,3 +69,11 @@ def load_model_tests(data_dir=os.path.join(DATA_DIR, 'model')):
             ModelTestCase(test_name, url, model_name, model_dir))
 
     return testcases
+
+
+def load_model_tests(data_dir=os.path.join(DATA_DIR, 'model')):
+    return _load_model_tests(data_dir)
+
+
+def load_generated_tests(data_dir=os.path.join(DATA_DIR, 'generated')):
```

Comment:
maybe generalize it a bit? we're going to have exported tests from various places and it might make sense to separate them down the road

Reply:
Good idea. :-)